packages/block-library/src/template-part/edit/index.js

```diff
@@ -91,31 +93,12 @@ export default function TemplatePartEdit( {
 
 	return (
 		<>
-			<InspectorControls>
```

Comment:
Nice call moving all this to another file.  Its definitely a bit more organized now.